src/PhappCommandBase.php

```diff
@@ -103,6 +105,10 @@ protected function getPhappEnviromentVariables($directory = './') {
     foreach ($finder as $file) {
       $dotenv = new Dotenv();
       $env_vars = array_replace($env_vars, $dotenv->parse(file_get_contents($file->getPathname()), $file->getPathname()));
+      // If current's directory manifest contains env variables, add them.
+      if ($manifest_env_vars = $manifest->getEnvironment()) {
+        $env_vars = array_merge($env_vars, $manifest_env_vars);
```

Comment:
If there is bot, dotenv should win. let's use array_replace better also, see above. no need to renumber things.